client/schema.go

```diff
@@ -31,36 +31,36 @@ func ValidSchema(schema string) bool {
 // GetSchemaFieldRegistry type switches field registries depending on the configured schema.
 // For example, IBM RACF has a custom LDAP schema so the password is stored in a different
 // attribute.
-func GetSchemaFieldRegistry(schema string, newPassword string) (map[*Field][]string, error) {
-	switch schema {
+func GetSchemaFieldRegistry(cfg *Config, newPassword string) (map[*Field][]string, error) {
```

Comment:
I was briefly thrown by this function changing signature but keeping the same name, but i think the name still works.